terraform/variables.tf

```diff
@@ -0,0 +1,115 @@
+variable "ACCESS_KEY" {
+    type=string
+}
+
+variable "SECRET_ACCESS_KEY" {
+    type=string
+}
+
+variable "REGION" {
+    type=string
+}
+
+variable "VPC" {
+    type=string
+}
+
+variable "SG_DB_NAME" {
+    type=string
+}
+
+variable "SG_DB_IDENTIFIER" {
+    type=string
+}
+
+variable "DB_NAME" {
+    type=string
+}
+
+variable "DB_USERNAME" {
+    type=string
+}
+
+variable "DB_PASSWORD" {
+    type=string
+}
+
+variable "DB_SUBNET_GROUP" {
+    type=string
+}
+
+variable "DB_IDENTIFIER" {
+    type=string
+}
+
+variable "DB_PORT" {
+    type=string
```

Comment:
Is the port passed as a string?

Reply:
Yep, if passed as a number it throw a weird error